### ISSUE TITLE
How to reproduce the success rate metric?

### ISSUE THREAD

**Issue by huamo555** [junha-l/dexter#3]
Hello author, I trained the model according to your training script, and the final success rate was 63.4%. Other metrics are shown in the image. The P-FID value is close to that reported in the paper. The success rate differs somewhat from what you reported. Could you please tell me what the problem might be?

<img width="1704" height="1279" alt="Image" src="https://github.com/user-attachments/assets/e6163a57-eba7-485e-8c6c-2522ae9b6796" />

**Comment by junha-l** [comment]
Hi @huamo555, thanks for the report. 

The cause seems to a data packaging issue on our end, now fixed. In some scenes with ≥10 grasps, grasp poses were misaligned with their contact-point annotations. Since DextER reasons over contacts, this hits SR more than P-FID. We've republished the corrected data, so if you pulled before 2026-07-22, delete your copy, re-download, and retrain:
```bash
hf download --repo-type dataset EunhaPark/project_dexter --local-dir datasets
```

Two training notes: 
- On DexGYS, we found reducing training steps from 100K to 50K gives comparable performance at half the cost.
- The effective batch size matters for SR. We trained on 8 GPUs, so scale per-GPU batch or grad accumulation to match if you use fewer:
```bash
accelerate launch --num_processes 8 --mixed_precision bf16 scripts/train.py experiment_name=dexter_bs8x8
```
Let us know if SR still doesn't match. 

As a side note, we've also released official checkpoints you can evaluate directly.
